### ISSUE TITLE
Should read credentials file instead of config file

### ISSUE THREAD

**Issue by sciurus** [lonelyplanet/aws-mfa#11]
Quoting https://github.com/aws/aws-cli/issues/1095#issuecomment-69965076 :

> The ~/.aws/config and the AWS_CONFIG_FILE are an AWS CLI only configuration mechanism. If you're only working with the AWS CLI, then you can still use these values. However if you're using other AWS SDKs, then you can just use the ~/.aws/credentials file. This is a fixed location that's used by all the AWS SDKs, and is the recommended file to use for interop.

**Comment by sciurus** [comment]
Hmm, we don't actually read from the ~/.aws/config file, do we? We just look for it to

- give us some assurance the user has configured the aws cli
-  find the correct directory to store our files in

Would if we changed `find_aws_config` to look for either one?

**Comment by kwent** [comment]
Yes we need to change `find_aws_config` to check for `.aws/credentials` instead. `.aws/config` can be optional. And so `aws-mfa` is stopping right away :(
